apps/analytics/views.py

```diff
@@ -141,14 +142,23 @@ def get_submission_count(request, challenge_pk, duration):
 )
 @authentication_classes((ExpiringTokenAuthentication,))
 def get_challenge_phase_submission_count_by_team(
-    request, challenge_pk, challenge_phase_pk
+    request, challenge_pk, challenge_phase_pk, version
```

Comment:
As here it can be either slug or challenge phase id so can you please rename the variable to something like `challenge_phase_pk_or_slug`